bot/exts/evergreen/trivia_quiz.py

```diff
@@ -111,23 +318,39 @@ async def quiz_game(self, ctx: commands.Context, category: str = None) -> None:
                         done_question.append(question_dict["id"])
                         break
 
-                q = question_dict["question"]
-                answer = question_dict["answer"]
+                if "dynamic_id" not in question_dict:
+                    q = question_dict["question"]
+                    answers = question_dict["answer"].split(", ")
+                else:
+                    q, answers = DYNAMIC_QUESTIONS_FORMAT_FUNCS[
```

Comment:
This is hard to read and confusing because of black's formatting, perhaps consider making fun into a variable and then calling that, rather than doing it all in one operation